container/src/services/container.service.ts

```diff
@@ -209,7 +232,21 @@ export class ContainerService {
                 });
                 break;
               case LuigiInternalMessageID.CHECK_PATH_EXISTS_REQUEST:
-                this.dispatchWithPayload(Events.CHECK_PATH_EXISTS_REQUEST, targetCnt, event, event.data.data);
+                this.dispatchWithPayload(
+                  Events.CHECK_PATH_EXISTS_REQUEST,
+                  targetCnt,
+                  event,
+                  event.data.data,
+                  (data: object) => {
+                    target.postMessage(
+                      {
+                        msg: LuigiInternalMessageID.SEND_PATH_EXISTS_ANSWER,
+                        data
```

Comment:
there is no need to expose the correlation id to the customer developer and educate him where to grab it from the payload and to put it as part of the callback data.
We know where it is in the internal payload, so extract it and set it fixed in the data object of the postmessage in callback